cegui/src/falagard/WidgetComponent.cpp

```diff
@@ -171,11 +171,18 @@ namespace CEGUI
 
     void WidgetComponent::removePropertyInitialiser(const String& name)
     {
-        for(PropertyInitialiserList::iterator i = d_propertyInitialisers.begin();
-                i < d_propertyInitialisers.end();
-                ++i)
-            if(i->getTargetPropertyName() == name)
-                d_propertyInitialisers.erase(i);
+        PropertyInitialiserList::iterator f = d_propertyInitialisers.begin();
+        PropertyInitialiserList::iterator const l = d_propertyInitialisers.end();
+        for(; f != l; ++f)
+            if(f->getTargetPropertyName() == name)
+                break;
+        if(f == l) return;
+        for(PropertyInitialiserList::iterator i = f; ++i != l;)
+            if(f->getTargetPropertyName() != name) {
+                *f = *i;
+                ++f;
+            }
```

Comment:
I can't read the above like this, the indentation seems wrong and please put brackets on new line and maybe also brackets on single lines if you cascade them

Reply:
Hopefully fixed. Assuming C++98 code (even though I need C++11 switch to compile it.)

Reply:
yes it is readable now, i assume u tested this sufficiently so i will merge it